src/briefcase/integrations/android_sdk.py

```diff
@@ -1144,14 +1144,19 @@ def has_booted(self):
                 f"Unable to determine if emulator {self.device} has booted."
             ) from e
 
-    def run(self, *arguments):
+    def run(self, *arguments, stealth=False):
```

Comment:
I think `quiet` would be a clearer name than `stealth`.

And Instead of repeating its default value 3 times in this file, how about making `pidof` and `pid_exists` take a `**kwargs` which they pass to `run`, and the same between `run` and `subprocess.check_output`?

Reply:
/me facepalms. I cycled through a bunch of names trying to come up with one that felt right, and `quiet` was sitting right there.

The reorganisation of the kwargs is also a good call; done.